github_pr.py

```diff
@@ -436,7 +439,8 @@ def main():
     parser.add_argument('--mergecomment', default=":shipit:", help='string to look for when checking comments for "shipit" approval, during MERGE only')
     parser.add_argument('--condition-non-owner-merger', action='store_true', help='stops owner from being able to apply merge comment')
     parser.add_argument('--condition-approved-mergers', default=None, nargs='+', help='list of usernames of approved mergers')
-    parser.add_argument('--condition-approved-mergers-file', default='./MAINTAINERS.txt', nargs='?', help='file of usernames of approved mergers')
```

Comment:
when there was no default previously, this worked as intended. Maybe we should just go back to not having the default set and if the flag is set, it uses it.